src/Mariuzzo/LaravelJsLocalization/Generators/LangJsGenerator.php

```diff
@@ -214,4 +204,43 @@ private function getVendorKey($key)
 
         return $keyParts[2] .'.'. $keyParts[1] . '::' . $keyParts[3];
     }
+
+    private function processFile(&$messages, $file, $namespace = null)
+    {
+        $pathName = $file->getRelativePathName();
+        $extension = $file->getExtension();
+        if ($extension != 'php' && $extension != 'json') {
```

Comment:
I'd change it to for readability
$supportedExtensions = ['json', 'php'];
!in_array($extension, $supportedExtensions)

I'm thinking laravel actually does this already somehow, maybe we should look into 
The `Illuminate\Translation\Translator` uses `Illuminate\Translation\FileLoader`to load/process files. Maybe we can use some of that logic